### DISCUSSION TITLE
What is the way to get compile time dependencies ?

### DISCUSSION THREAD

**Discussion by NeerajGahlawatJava** [Keyfactor/ejbca-ce#511] [Q&A]
What is the way to get EJBCA **compile time dependencies** ?  and How to enable tls1.2 ?

**Comment by primetomas** [comment]
Not sure what you mean with compile time dependencies? You have the installation pre-requisites here:
https://doc.primekey.com/ejbca/ejbca-installation/installation-prerequisites


For TLS that is configured in the application server, i.e. in the installation instructions TLS 1.2 and 1.3 is added by default, but you can re-configure it how you want:
https://doc.primekey.com/ejbca/ejbca-installation/application-servers/wildfly-26#WildFly26-HTTP(S)Configuration
(in the container I guess 1.2 is enabled by default)

**Comment by miniwalia** [comment]
We need dependency tree of all the jars used by EJBCA
For example, in maven we use mvn dependency:tree, gradle dependencies
How to get similar dependency tree for EJBCA?

**Reply by primetomas** [reply]
It's here: https://github.com/Keyfactor/ejbca-ce

**Reply by miniwalia** [reply]
Thanks for providing link. 
I have gone through the documentation mentioned above but could not find command for ant build with dependency tree generation.It would be helpful if you can point me to the file or command which I should refer

**Reply by miniwalia** [reply]
Do you have guidance on configuring ivy setting with EJBCA
